mode/smalltalk/smalltalk.js

```diff
@@ -15,6 +15,7 @@ CodeMirror.defineMode('smalltalk', function(config) {
 
   var specialChars = /[+\-\/\\*~<>=@%|&?!.,:;^]/;
   var keywords = /true|false|nil|self|super|thisContext/;
+  var variables = /Smalltalk/;
```

Comment:
I feel like I'm in a lecture! heh. I will make a note of this and a) fix and b) not do it again!